Objects/codeobject.c

```diff
@@ -1924,3 +1924,15 @@ _PyStaticCode_Dealloc(PyCodeObject *co)
         co->co_weakreflist = NULL;
     }
 }
+
+void
+_PyStaticCode_InternStrings(PyCodeObject *co) 
+{
+    int res = intern_strings(co->co_names);
+    assert(res == 0);
+    res = intern_string_constants(co->co_consts, NULL);
+    assert(res == 0);
+    res = intern_strings(co->co_localsplusnames);
+    assert(res == 0);
+    (void)res;
+}
```

Comment:
Hm, I'm not comfortable with ignoring errors here. In theory this could run out of memory in intern_string_constants(). The function should just return 0 for success and -1 if any of the calls fail, and the caller should handle it.

(TBH, unmarshal_frozen_code() currently doesn't handle errors from get_code(), but IMO it should. It handles errors from other calls.)

Reply:
I don't think so because in `intern_strings`, it only checks if all the items of the tuple are strings and it doesn't check if `PyUnicode_InternInPlace` fails, moreover `PyUnicode_InternInPlace`  itself ignores memory errors during interning strings so `assert` is enough.

`intern_strings`:
https://github.com/python/cpython/blob/8e98175a03fe03d62822d96007a74e5273013764/Objects/codeobject.c#L37-L50

`PyUnicode_InternInPlace`:
https://github.com/python/cpython/blob/8e98175a03fe03d62822d96007a74e5273013764/Objects/unicodeobject.c#L15615-L15627